src/routes/InventoryViews.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { PageSection, Title } from '@patternfly/react-core';
-import SystemsDataViewTable from '../components/SystemsDataViewTable';
+import SystemsDataViewTable from '../components/SystemsViewTable';
```

Comment:
I'd change the import name as well to keep the same name for the component through the codebase.
```suggestion
import SystemsViewTable from '../components/SystemsViewTable';
```